src/core/server/saved_objects/import/resolve_import_errors.ts

```diff
@@ -131,6 +131,7 @@ export async function resolveSavedObjectsImportErrors({
     retries,
     createNewCopies,
     dataSourceId,
+    workspaces,
   };
   const checkConflictsResult = await checkConflicts(checkConflictsParams);
```

Comment:
how would workspaces checked in checkConflicts?

Reply:
checkConflicts logic: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/workspace/server/saved_objects/saved_objects_wrapper_for_check_workspace_conflict.ts#L219